udata/api/__init__.py

```diff
@@ -278,6 +281,44 @@ class API(Resource):  # Avoid name collision as resource is a core model
     pass
 
 
+oembed_parser = api.parser()
+oembed_parser.add_argument(
+    'url', type=str, help='URL of the resource to embed.', required=True)
```

Comment:
`from flask_restul import inputs`
then you can write `type=inputs.url` and it will perform validation